src/hotspot/share/nmt/memBaseline.hpp

```diff
@@ -141,7 +141,7 @@ class MemBaseline {
   size_t malloc_tracking_overhead() const {
     assert(baseline_type() != Not_baselined, "Not yet baselined");
     MemBaseline* bl = const_cast<MemBaseline*>(this);
-    return bl->_malloc_memory_snapshot.malloc_overhead();
+    return bl->_malloc_memory_snapshot.nmt_overhead();
```

Comment:
Better name for 'nmt_overhead()` would be `malloc_tracking_overhead()` to be similar to its caller as here.